Source/WebCore/platform/graphics/TextRun.h

```diff
@@ -128,7 +128,7 @@ class TextRun {
     float m_horizontalGlyphStretch;
 
     float m_expansion;
-    unsigned m_expansionBehavior : 4;
+    ExpansionBehavior m_expansionBehavior;
```

Comment:
FYI: This will increase the size of TextRun. It's probably fine, though, because there should only ever be a single TextRun alive at a time.

Reply:
I'll change it so `ExpansionBehavior` would use bitfields for members to reduce it to one byte. The size of a `TextRun` should not change then, because the old `TextRun` has bitfields totaling to 9 bits = 2 bytes. The new TextRun has 1 byte from `ExpansionBehavior` + 5 bits = still 2 bytes. Other fields' size doesn't change.